sdk/src/Core/Amazon.Util/AWSSDKUtils.cs

```diff
@@ -469,6 +475,30 @@ public static string CanonicalizeResourcePathV2(Uri endpoint, string resourcePat
             return canonicalizedResourcePath;
         }
 
+        /// <summary>
+        /// This method returns the full canonicalized resource path which includes the resource path from the request and the resource path from the endpoint.
+        /// This is just a helper method to be used internally.
```

Comment:
That fact that this is a `private static` method is enough to say this is "just a helper method".